specs/tendrl_host_listing_summary_api.adoc

```diff
@@ -0,0 +1,187 @@
+// vim: tw=79
+
+= Host Listing Backend Support
+
+== Introduction
+
+The host listing page displays the following details per host:
+
+* Hostname and status
+* Storage used, total and percentage used
+* Cpu percentage used
+* Memory used, total and percentage used
+* Name of Cluster to which host belongs
+* Role of the node in the cluster
```

Comment:
@shtripat this spec is meant only for capturing details about monitoring section(The title of the spec is little misleading I'll change it :) ) The section in introduction just starts by listing all details that the host listing page in UI requires and then goes onto detail the monitoring sections of it